src/core/server/http/router/request.ts

```diff
@@ -92,4 +106,15 @@ export class KibanaRequest<Params = unknown, Query = unknown, Body = unknown> {
   public unstable_getIncomingMessage() {
     return this.request.raw.req;
   }
+
+  private getRouteInfo() {
+    return {
+      path: this.request.path,
+      method: this.request.method,
+      options: {
+        authRequired: this.request.route.settings.auth !== false,
```

Comment:
If `auth` is always a boolean, could this instead be:

```js
authRequired: this.request.route.settings.auth,
```

Reply:
- it's not boolean. `false | string | RouteOptionsAccess;` 
- we invert the logic. authRequired always if not specified `false` explicitly